include/robot_interfaces/n_joint_robot_types.hpp

```diff
@@ -161,11 +196,36 @@ struct NJointRobotTypes
         }
     };
 
-    struct Observation
+    struct Observation : public Loggable
     {
         Vector position;
         Vector velocity;
         Vector torque;
+
+        std::vector<std::string> get_name() override
+        {
+            return {"Position", "Velocity", "Torque"};
+        }
+
+        std::vector<std::vector<double>> get_data() override
+        {
+            std::vector<double> position_;
+            position_.resize(position.size());
+            Vector::Map(&position_[0], torque.size()) = position;
```

Comment:
This uses `torques.size()` for the position.

Reply:
Omg, thanks. Fixed.